doc/chomp_interface/chomp_interface_tutorial.rst

```diff
@@ -49,10 +49,81 @@ Using CHOMP with Your Robot
       <arg name="debug" value="$(arg debug)"/>
       <arg name="planner" value="chomp" />
     </include>
-#. Open the ``move_group.launch`` file in your ``<robot_moveit_config>/launch/`` folder and make two changes. First, add ``<arg name="planner" default="ompl" />`` just under the ``<launch>`` tag and second, within the ``<include ns="move_group">`` tag replace ``<arg name="pipeline" value="ompl" />`` with ``<arg name="pipeline" value="$(arg planner)" />``.
+#. Open the ``move_group.launch`` file in your ``<robot_moveit_config>/launch/`` folder and make two changes.
+
+ 7.1. First, add ``<arg name="planner" default="ompl" />`` just under the ``<launch>`` tag, and,
+
+ 7.2. Second, within the ``<include ns="move_group">`` tag replace ``<arg name="pipeline" value="ompl" />`` with ``<arg name="pipeline" value="$(arg planner)" />``.
 
 Running the Demo
 ----------------
 If you have the ``panda_moveit_config`` from the `ros-planning/panda_moveit_config <https://github.com/ros-planning/panda_moveit_config>`_ repository you should be able to simply run the demo: ::
 
-  roslaunch panda_moveit_config demo_chomp.launch
\ No newline at end of file
+  roslaunch panda_moveit_config demo_chomp.launch
+
+Testing CHOMP with Obstacles in the Scene
+-----------------------------------------
+To test CHOMP in an evironment with obstacles, you can run any of the sample python scripts 
```

Comment:
end sentence with ``: ::``

Reply:
if I end this sentence with : :: the formatting for the bullets below this line gets messed up like: 
![selection_024](https://user-images.githubusercontent.com/5405862/41138214-b875d626-6aad-11e8-83c6-59d974c1ab75.png)

Currently I just end it with :